hw7_Zhurylo/js/index.js

```diff
@@ -0,0 +1,11 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const firstNumber = +prompt('Enter the first number')
+    const secondNumber = +prompt('Enter the second number')
+    const thirdNumber = +prompt('Enter the third number')
+    if ( (isNaN(firstNumber)) || (isNaN(secondNumber)) || (isNaN(thirdNumber))) {
```

Comment:
brackets is redundant
```suggestion
    if( isNaN(firstNumber) || isNaN(secondNumber) || isNaN(thirdNumber)) {
```